packages/workers-shared/asset-worker/tests/handler.test.ts

```diff
@@ -1,9 +1,13 @@
 import { vi } from "vitest";
 import { mockJaegerBinding } from "../../utils/tracing";
 import { applyConfigurationDefaults } from "../src/configuration";
-import { handleRequest } from "../src/handler";
+import { canFetch, handleRequest } from "../src/handler";
 import type { AssetConfig } from "../../utils/types";
 
+const mockEnv = {
+	JAEGER: mockJaegerBinding(),
+};
```

Comment:
Given that the production code has to do a check for a missing JAEGER binding anyway, is this needed?